test.js

```diff
@@ -1681,3 +1681,119 @@ test('fastify.view with handlebars engine and callback in production mode', t =>
     })
   })
 })
+
+test('reply.view with art-template engine and custom templates folder', t => {
+  t.plan(6)
+  const fastify = Fastify()
+  const art = require('art-template')
+  const data = { text: 'text' }
+
+  fastify.register(require('./index'), {
+    engine: {
+      'art-template': art
+    },
+    templates: 'templates'
+  })
+
+  fastify.get('/', (req, reply) => {
+    reply.view('./index.art', data)
+  })
+
+  fastify.listen(10086, err => {
+    t.error(err)
+
+    sget({
+      method: 'GET',
+      url: 'http://127.0.0.1:10086/'
+    }, (err, response, body) => {
+      t.error(err)
+      t.strictEqual(response.statusCode, 200)
+      t.strictEqual(response.headers['content-length'], '' + body.length)
+      t.strictEqual(response.headers['content-type'], 'text/html; charset=utf-8')
+
+      art.defaults.root = './temlates'
```

Comment:
There are still the typo for all the `art.defaults.root`
Since the test are passing, is this line needed?

Reply:
fix

Reply:
fix